components/Header/Header.tsx

```diff
@@ -8,11 +8,13 @@ import Search from "../Search/Search";
 import { Actions, ResponsiveActions } from "./Header.styled";
 import { HamburgerMenuIcon } from "@radix-ui/react-icons";
 import { useRouter } from "next/router";
+import env from "@/.canopy/env.json"
```

Comment:
I don't think we need this `env.json`. Next has some built in env stuff we can latch onto. It's ugly for now and we should build helpers but this is what it looks like:

```suggestion

const mapEnabled = process.env.CANOPY_CONFIG.map.enabled;
```